tests/os_snapshot_test/os_snapshot_windows_test.go

```diff
@@ -21,29 +24,27 @@ func TestShadowCopy(t *testing.T) {
 	}
 
 	runner := testenv.NewExeRunnerWithBinary(t, kopiaExe)
-
 	e := testenv.NewCLITest(t, testenv.RepoFormatNotImportant, runner)
+
 	e.RunAndExpectSuccess(t, "repo", "create", "filesystem", "--path", e.RepoDir)
+	e.RunAndExpectSuccess(t, "policy", "set", "--global", "--enable-volume-shadow-copy=when-available")
```

Comment:
relocated here from below, to setup the "repo" first, no functional change to the test.